virttest/utils_hotplug.py

```diff
@@ -46,6 +56,7 @@ def create_mem_xml(tg_size, pg_size=None, mem_addr=None, tg_sizeunit="KiB",
     :param uuid: Value of attrib 'uuid' of memory
     :return: Returns a copy of Memory Hotplug xml instance.
     """
+    LOG.warn("This function is deprecated now. See more details in docstring")
```

Comment:
The 'warn' method is deprecated, could you  update to use 'warning'?

Reply:
updated